SmartDeviceLink/SDLChoiceSetManager.m

```diff
@@ -353,8 +391,15 @@ - (void)presentChoiceSet:(SDLChoiceSet *)choiceSet mode:(SDLInteractionMode)mode
             [strongOp.choiceSet.delegate choiceSet:strongOp.choiceSet didSelectChoice:strongOp.selectedCell withSource:strongOp.selectedTriggerSource atRowIndex:strongOp.selectedCellRow];
         }
 
-        weakself.pendingPresentationSet = nil;
-        weakself.pendingPresentOperation = nil;
+        strongSelf.pendingPresentOperation = nil;
+
+        // Check if the manager has shutdown because the list of pending choices should not be updated
+        if ([strongSelf.currentState isEqualToString:SDLChoiceManagerStateShutdown]) {
+            SDLLogD(@"The manager has shutdown");
+            return;
+        }
+
+        strongSelf.pendingPresentationSet = nil;
```

Comment:
But the `pendingPresentationSet` can be sent to nil anyway, right? So there's no reason to add the shutdown check? It doesn't seem to anything except skipping the `pendingPresentationSet`.

Reply:
Fixed. Removed the state check.